doc/languages-frameworks/agda.section.md

```diff
@@ -193,42 +192,19 @@ To add an Agda package to `nixpkgs`, the derivation should be written to `pkgs/d
 { }
 ```
 
+You can look at other files under `pkgs/development/libraries/agda/` for inspiration.
+
 Note that the derivation function is called with `mkDerivation` set to `agdaPackages.mkDerivation`, therefore you
 could use a similar set as in your `default.nix` from [Writing Agda Packages](#writing-agda-packages) with
 `agdaPackages.mkDerivation` replaced with `mkDerivation`.
 
-Here is an example skeleton derivation for iowa-stdlib:
-
-```nix
-mkDerivation {
-  version = "1.5.0";
-  pname = "iowa-stdlib";
-
-  src = <...>;
-
-  libraryFile = "";
-  libraryName = "IAL-1.3";
-
-  buildPhase = ''
-    runHook preBuild
-
-    patchShebangs find-deps.sh
-    make
-
-    runHook postBuild
-  '';
-}
-```
-
-This library has a file called `.agda-lib`, and so we give an empty string to `libraryFile` as nothing precedes `.agda-lib` in the filename. This file contains `name: IAL-1.3`, and so we let `libraryName =  "IAL-1.3"`. This library does not use an `Everything.agda` file and instead has a Makefile, so there is no need to set `everythingFile` and we set a custom `buildPhase`.
-
 When writing an Agda package it is essential to make sure that no `.agda-lib` file gets added to the store as a single file (for example by using `writeText`). This causes Agda to think that the nix store is a Agda library and it will attempt to write to it whenever it typechecks something. See [https://github.com/agda/agda/issues/4613](https://github.com/agda/agda/issues/4613).
 
 In the pull request adding this library,
 you can test whether it builds correctly by writing in a comment:
 
 ```
-@ofborg build agdaPackages.iowa-stdlib
+@ofborg build agdaPackages.library-name
```

Comment:
Nit: is there some way to make it clear that `library-name` should be substituted for the actual package name. (Maybe this is really obvious and I'm nitpicking over nothing)